src/test/java/com/github/ermay12/regex/PhoneNumberClientCode.java

```diff
@@ -26,5 +27,13 @@ public void testPhoneNumber() {
                                     capture(DIGIT),
```

Comment:
can you get rid of all the unused captures?

Reply:
They're not supposed to be redundant - you should be able to do something like `result.get().group(i)` to get the i'th digit in the phone number

Reply:
so for instance you can do .group(1 - 3) to get the area code